File: src/test/java/com/csl/PersonTest.java
package com.csl;

import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

//測試檔案命名規則 = 物件名稱後加Test
public class PersonTest {

    //測試方法命名規則 = 方法名稱後加Test or isCorrect
    @Test
    public void bmiTest() {
        Person person = new Person(63, 1.7f);
        Assertions.assertEquals( 63/1.7f/1.7f , person.bmi());
    }
}

File: src/test/kotlin/com/csl/PersonTest.kt
package com.csl.kotlin

import com.csl.Human
import com.csl.Person
import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test



class PersonTest {

    //可以用kotlin寫java的測試程式
    @Test
    fun personBmiTest() {
        var person:Person = Person(62f, 1.7f)
        Assertions.assertEquals( 62f/1.7f/1.7f, person.bmi())
    }

    @Test
    fun humanBmiTest() {
        var human:Human = Human("Chang", 62f, 1.7f)
        Assertions.assertEquals(62f/1.7f/1.7f, human.bmi())
    }
}